packages/vue/src/types/index.ts

```diff
@@ -67,7 +70,8 @@ export interface ConfirmPasswordSetupReturnTypes {
   confirmText: ComputedRef<string>;
   onLostCodeClicked: () => void;
   state: Ref;
-  send: PayloadSender<EventObject>;
+  actorState: Ref;
```

Comment:
This probably should be `ComputedRef` . If you wanted to be more specific you'd need to add in some type assertion by defining what it will return inside the computed property, otherwise it defaults to `ComputedRef<any>`. 

To fix this, you'd need to go back into `getActorState` and add a return type in there, then add the type in this file.

Reply:
Done! Thanks for the explanation.